### PR TITLE
Issue 62: Ran Parquet Files

### LINKED ISSUE #62: Feature stitching
# **Feature Summary**
A file that takes all the scripts, libraries, etc. and calls them in a way that the end user can execute the application simply and easily.

## **Objective**
Stitch all functions together to get something presentable for client

## **Key Requirements**
- [ ] Work properly
- [ ] Work quickly
- [ ] No duplicates/minimal redundant code being called

## **Suggested Implementation**
Outline the basic approach for adding this feature.

## **Dependencies**
List any dependencies or prerequisites.

## **Additional Notes**
Any other relevant details or considerations.

### PR DESCRIPTION
**Pull request by zazwaz12**
---
Title: "Closes #62 "
---

# **Summary of Changes**
The file for converting Files to Parquet.

# **Checklist:**
- [x] My code follows the code style of this project.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests prove my fix is effective or that my feature works.
- [x] I have annotated added types appropriately.

### CODE REVIEW

**Review by ctmes — APPROVED** [review]
Looks great

File: scripts/spreadsheet_to_parquet.py [hunk 1]
```diff
@@ -3,63 +3,75 @@
...
+# Utility functions for file listing and reading spreadsheets
 list_files = nhs.utils.path.list_files
 get_reader = nhs.data.handling.get_spreadsheet_reader
 
 
 @logger.catch()
 def save_parquet(path: str, input_dir: str, output_dir: str):
+    # Define relative output file path
     relative_path = Path(path).relative_to(input_dir)
+    output_dir = input_dir.replace("FilesIn", "AppStaging")
```
Comment (SodaVolcano) [inline scripts/spreadsheet_to_parquet.py:23]: remove

File: scripts/spreadsheet_to_parquet.py [hunk 2]
```diff
@@ -3,63 +3,75 @@
...
 
+    # Skip if file already exists
     if output_file_path.exists():
         logger.warning(f"File {output_file_path} already exists. Skipping.")
         return
 
+    if "Standard" in str(path) and "DEFAULT_GEOCODE" not in str(path):
+        logger.warning(f"Only taking DEFAULT GEOCODE from GNAF for now. Skipping.")
+        return
+
```
Comment (SodaVolcano) [inline scripts/spreadsheet_to_parquet.py:34]: remove

**Review by SodaVolcano — CHANGES_REQUESTED** [review]

**Review by SodaVolcano — APPROVED** [review]

**Comment by SodaVolcano** [comment]
FYI I've extended the help message

**Pull request merged**